runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -646,13 +704,13 @@ def TermDebugSendCommand(cmd: string)
     ch_sendraw(gdb_channel, $"{cmd}\n")
   else
     var do_continue = 0
-    if !stopped
+    if stopped == false
```

Comment:
When using a Boolean variable, there is no need to it compare against true or false. The previous code is more readable.

Reply:
Done! I extended the process also in non-cited places that suffer of the same issue.